spec/admin.mediawiki

```diff
@@ -32,13 +32,19 @@ possible.
 {|
 ! variable !! units !! description
 |-
-| lot&nbsp;size  || atoms ||  the minimum order quantity and the order quantity increment when an asset is the base asset
+| lotSize  || int ||  the minimum order quantity and the order quantity increment when an asset is the base asset
 |-
-| rate&nbsp;step || atoms || the minimum price rate and the price rate increment when an asset is the quote asset. [[orders.mediawiki/#Rate_Encoding|message-rate encoding]]
+| rateStep || int || the minimum price rate and the price rate increment when an asset is the quote asset. [[comm.mediawiki/#Rate_Encoding|message-rate encoding]]
 |-
-| fee&nbsp;rate  || atoms/byte || the minimum fee rate for swap transactions
+| swapConf || int || the Minimum confirmations before acting on a swap transaction
 |-
-| swap&nbsp;confirmations|| count || the Minimum confirmations before acting on a swap transaction
+| maxFeeRate  || int || the maximum fee rate for swap transactions
+|-
+| network || string || the network the coin daemon is running one (i.e. "mainnet")
+|-
+| bip44symbol || string || the symbol of the [https://github.com/satoshilabs/slips/blob/master/slip-0044.md registered coin type]
+|-
+| configPath || string || the path to the coin daemon's config file or ipc file in the case of Ethereum
```

Comment:
I'd say these three are not necessary. The network is not a **per-asset variable**. The symbol is not really a variable either.

Reply:
Then lets get rid of these tables entirely.  This page isn't describing the asset and market variables.  Those are covered elsewhere in the spec.